### PR TITLE
Pkg.ocb_tags

### PR DESCRIPTION
**Pull request by pqwy**
Adds a helper converting build configuration keys into `ocamlbuild` tags.

### CODE REVIEW

**Comment by dbuenzli** [comment]
I think this would benefit in being a bit more general.

**Comment by dbuenzli** [comment]
Ping.

**Comment by pqwy** [comment]
`ocb_tag` could have `string -> string` instead of forcing the format of the tag to `name(value)`, but the more general behavior can be recovered with a bit more code and I expect this to be the main use.

`pp_value` could be made public.

`ocb_tag_b` sounds contrived until you need to type `ocb_boolean_tag`.

**Comment by dbuenzli** [comment]
> ocb_tag_b sounds contrived until you need to type ocb_boolean_tag.

It doesn't sound contrived, it *reads* ugly and cryptic. What about `ocb_bool_tag[s]` ?

**Comment by pqwy** [comment]
I _did_ foresee that would be the main objection -- pick any name, optimizing for length.

What about the rest?

File: src/topkg_build.ml [hunk 1]
```diff
@@ -62,6 +62,17 @@ let codec =
   Topkg_codec.version 0 @@
   Topkg_codec.(view ~kind:"build" fields (pair prepare_on_pin dir))
 
+let ocb_tag c v tag =
+  Format.(fprintf str_formatter) "%s(%a)" tag (Topkg_conf.pp_value c) v;
```
Comment (dbuenzli) [inline src/topkg_build.ml]: Please use `Topkg_string.strf` here.

**Comment by dbuenzli** [comment]
Thanks your patch is in as 2776861dd0bcdf14e3770f5 further tweaked by db21657b8010b8618c

**Pull request closed without merging**